client-node.py

```diff
@@ -63,6 +77,7 @@
 else:
     log.startLogging(sys.stdout)
 
+'''
```

Comment:
Basically the ''' turns anything after into a string. That's how you block comment in python so
'''
print("hi")
shutil.rmtree("/")
'''
would be commented out. You can also use it as a multi-line string in print()